addons/medical/functions/fnc_addPain.sqf

```diff
@@ -0,0 +1,28 @@
+/*
+ * Author: commy2
+ * Adds pain to unit and plays a scream sound.
```

Comment:
This function can currently also reduce pain.

Also, there is an upper limit but no lower limit enforced here.